demos/focused_api_test.py

```
@@ -0,0 +1,199 @@
+        else:
+            print("  ❌ Answer extraction failed")
+            
+    except Exception as e:
+        print(f"  ❌ Error: {e}")
+
+async def main():
+    """Run focused tests."""
+    
+    try:
+        # Test 1: Basic QADI functionality
+        qadi_result = await test_template_vs_llm_comparison()
```

Ruff (0.11.9)

Local variable `qadi_result` is assigned to but never used

Remove assignment to unused variable `qadi_result`

(F841)